spec/system/task_spec.rb

```diff
@@ -43,11 +53,12 @@
   end
 
   describe 'Task詳細' do
+    let(:task) { FactoryBot.create(:task, project_id: project.id) }
```

Comment:
設定ファイルに追記すると省略して記載できます。
```suggestion
    let(:task) { create(:task, project_id: project.id) }
```

Reply:
6831bed で`FactoryBot.`を省略しました。（設定ファイルは記載済みでした）